cmd/soroban-cli/src/commands/contract/build.rs

```diff
@@ -145,6 +147,28 @@ impl Cmd {
                     return Err(Error::Exit(status));
                 }
 
+                let file = format!("{}.wasm", p.name.replace('-', "_"));
+                let target_file_path = Path::new(target_dir)
+                    .join("wasm32-unknown-unknown")
+                    .join(&self.profile)
+                    .join(&file);
```

Comment:
Since `file` and `target_file_path` are now defined here, they no longer need to be defined on lines 175-179. The same definition appears there and is a duplicate.

Reply:
updated in [344378a](https://github.com/stellar/stellar-cli/pull/1684/commits/344378a3f68ec0adf39de08906391bf046ea5b56)